torch/csrc/jit/passes/quantization.cpp

```diff
@@ -215,6 +242,15 @@ void InsertQuantDequantNodes(std::shared_ptr<Graph>& graph) {
   std::vector<Value*> quantOutputs;
   std::unordered_set<Value*> valueLookup;
 
+  // Observer nodes to remove from graph
+  std::vector<Node*> nodes_to_remove;
+
+  // Create value:qparam dict. Even if ssa-name changed between
```

Comment:
While I understand what you're trying to say here, I don't think that message reflects what's going on in the code. The dictionary we create here has nothing to do with guarantees about correct mapping, it's just a convenience thing at this point. All guarantees were provided at the time of construction of observer nodes and the input dictionary. Could you think of a better wording here please?